src/Parser.js

```diff
@@ -977,11 +980,12 @@ export default class Parser {
         // Transform combining characters into accents
         if (match) {
             for (let i = 0; i < match[0].length; i++) {
-                const accent = match[0][i];
+                const accent: string = match[0][i];
                 if (!unicodeAccents[accent]) {
                     throw new ParseError(`Unknown accent ' ${accent}'`, nucleus);
                 }
-                const command = unicodeAccents[accent][this.mode];
+                const command: ?{text: string, math?: string} =
+                    unicodeAccents[accent][this.mode];
```

Comment:
Is `unicodeAccents` not typed in such a way that the proper type for `command` would be inferred?

Reply:
There are no flow types on `unicodeAccents` it since it's also used by `unicodeMake`. So I unfortunately had to do it this way.

Reply:
This was wrong. I no longer needed to add the type. I'd misread the original double-nested map lookup. I ran into this while working on the second PR.